lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -292,6 +281,9 @@ class _ZoomableImageState extends State<ZoomableImage>
         initialPosition: newPosition,
         initialScale: scale,
       );
+      setState(() {
```

Comment:
This method (`_updatePhotoViewController`) is only called after the final image has been loaded. In case of images that don't exist locally, this could take a while. So the `initialScale` will be `null` until then, affecting this whole behavior

Reply:
oh I see

Reply:
So, while loading, what will be shown then? @vishnukvmd

Reply:
Ref. `PhotoViewScaleState.initial`

Reply:
initialScale null case is also updated